pkg/scheduler/tests/scheduler_smoke_test.go

```diff
@@ -936,3 +938,136 @@ partitions:
 
     waitForAcceptedApplications(mockRM, "app-added-2", 1000)
 }
+
+func TestSchedulingOverCapacity(t *testing.T) {
+    serviceContext := entrypoint.StartAllServicesWithManualScheduler()
+    defer serviceContext.StopAll()
+    proxy := serviceContext.RMProxy
+
+    // Register RM
+    configData := `
+partitions:
+  -
+    name: default
+    queues:
+      -
+        name: root
+        submitacl: "*"
+        queues:
+          -
+            name: default
+            properties:
+              application.sort.policy: fair
+            resources:
+              guaranteed:
+                memory: 100
+                vcore: 10
+              max:
+                memory: 100
+                vcore: 10
+`
+    configs.MockSchedulerConfigByData([]byte(configData))
+    mockRM := NewMockRMCallbackHandler(t)
+
+    _, err := proxy.RegisterResourceManager(
+        &si.RegisterResourceManagerRequest{
+            RmId:        "rm:123",
+            PolicyGroup: "policygroup",
+            Version:     "0.0.2",
+        }, mockRM)
+
+    if err != nil {
+        t.Error(err.Error())
+    }
+
+    // Register a node, and add applications
+    err = proxy.Update(&si.UpdateRequest{
+        NewSchedulableNodes: []*si.NewNodeInfo{
+            {
+                NodeId: "node-1:1234",
+                Attributes: map[string]string{
+                    "si.io/hostname": "node-1",
+                    "si.io/rackname": "rack-1",
+                },
+                SchedulableResource: &si.Resource{
+                    Resources: map[string]*si.Quantity{
+                        "memory": {Value: 100},
+                        "vcore":  {Value: 10},
```

Comment:
To make sure that we do not get limited by the node we should do one of two things:
- decrease the queue size to 50MB/5vcore and ask for 70/7
or
- increase the node to 150MB/15vcores and leave the ask

In the case we have now even if we regress in the queue area the node size will prevent the test from failing.

Reply:
Done